apps/judicial-system/backend/src/app/modules/case/filters/case.filters.ts

```diff
@@ -328,12 +328,29 @@ export function getCasesQueryFilter(user: User): WhereOptions {
         ]
       : []
 
+  const blockDraftIndictmentsForCourt = [
```

Comment:
This look fine. However, this only excludes from the returned list of cases. It does not block direct access to individual cases. To block direct access, you would need to modify the function `isCaseBlockedFromUsera`.

You could accomplish blocking for both case list and direct access in one go, by passing the case type to `getAllowedStates`. I think that would be a much simpler change and would keep the change in one place.

Reply:
It looks like that is what we want. [See the description in asana](https://app.asana.com/0/1199153462262248/1202978837144009/f):  

> Ekki öryggis-issue heldur bara filtering í bakenda.
> R-mál eiga að sjást áfram á vinnslustigi en S-mál eiga ekki að sjást í listanum fyrr en þau eru komin með stöðuna Send (ákærandi er búinn að senda á dómstólinn)

Reply:
I think the wording in the ticket was intended to make the implementation easier, i.e. we don't have to block direct access if it complicates matters. However, I feel the code is more complicated if we don't block both and it may cause confusion down the road if case list and direct access blocking are not identical.

However, I will accept this as is, but please consider making the two behave in the same way.

Reply:
Addressed in https://github.com/island-is/island.is/pull/8861/commits/a0135471f0d4fdb5f8bef31a71977ed2c74c392b